iocore/net/P_UnixNet.h

```diff
@@ -389,30 +400,171 @@ check_transient_accept_error(int res)
   }
 }
 
+static inline void
+net_activity(UnixNetVConnection *vc, EThread *thread)
+{
+  Debug("socket", "net_activity updating inactivity %" PRId64 ", NetVC=%p", vc->inactivity_timeout_in, vc);
+  (void)thread;
+  if (vc->inactivity_timeout_in) {
+    vc->next_inactivity_timeout_at = Thread::get_hrtime() + vc->inactivity_timeout_in;
+  } else {
+    vc->next_inactivity_timeout_at = 0;
+  }
+}
+
+//
+// Reschedule a UnixNetVConnection by moving it
+// onto or off of the ready_list
+//
+TS_INLINE void
+NetHandler::read_reschedule(UnixNetVConnection *vc)
+{
+  vc->ep.refresh(EVENTIO_READ);
+  if (vc->read.triggered && vc->read.enabled) {
+    read_ready_list.in_or_enqueue(vc);
+  } else {
+    read_ready_list.remove(vc);
+  }
+}
+
+TS_INLINE void
+NetHandler::write_reschedule(UnixNetVConnection *vc)
+{
+  vc->ep.refresh(EVENTIO_WRITE);
+  if (vc->write.triggered && vc->write.enabled) {
+    write_ready_list.in_or_enqueue(vc);
+  } else {
+    write_ready_list.remove(vc);
+  }
+}
+
+//
+// Signal an event
+//
+TS_INLINE int
+NetHandler::read_signal_and_update(int event, UnixNetVConnection *vc)
```

Comment:
Do we want to have more consistency by always having the `UnixNetVConnection*` argument first?